features/provider/admin/account/authentication_providers/new.feature

```diff
@@ -45,9 +45,9 @@ Feature: Account Settings > Users > SSO Integrations > New
   Scenario: Realm or Site hint is accurate
     Given they go to the new sso integration page
     When they select "Red Hat Single Sign-On" from "SSO Provider"
-    Then they should see "Ex: https://rh-sso.example.com/auth/realms/demo"
+    Then they should see "e.g. https://rh-sso.example.com/auth/realms/demo"
```

Comment:
I googled, and it seems that `e.g.` is the more conventional way to provide an example, while `Ex.` is not commonly used.

In fact, it seems to be more correct to add a comma after - `e.g.,` but I think it looks clumsy :grimacing: